Walkthrough Guide/07 API Management/README.md

```diff
@@ -132,21 +132,15 @@ We can go ahead and click ***Save***, and API Management is ready for testing.
 
 #### 3.1.2 Jobs
 
-To save time, we've produced API specifications for you to use with this project which allow API Management to automatically configure your APIs and operations. You can find these in the [`/Swagger`](/Assets/Swagger/) folder above.
-
-You'll want to edit these on [Line 7](https://github.com/MikeCodesDotNet/Mobile-Cloud-Workshop/blob/cae3c1e5366a78170eb217f897b7d4398f7bfd32/Walkthrough%20Guide/05_API_Management/Assets/Swagger/Jobs.swagger.json#L7) to point to your App Service instance rather than ours.
-
-![Search for API Management](Assets/EditingSwaggerDEf.png)
-
-Change Line 7 to your service and save.
+To save time, we will use the API specifications generated by Swagger which allow API Management to automatically configure your APIs and operations. You can find these in the {root uri of your service}/swagger.
```

Comment:
Can we replace the last sentence with "You can find these at the `/swagger` endpoint of your Web App. In our case, that's `http://myawesomestartupapi.azurewebsites.net/swagger`."

That's what we used at most other places in the docs.